packages/ui/src/multiplying-architecture/KaotoEditorChannelApi.ts

```diff
@@ -81,4 +82,11 @@ export interface KaotoEditorChannelApi extends KogitoEditorChannelApi {
   getRuntimeInfoFromMavenContext(): Promise<
     CamelMainMavenInformation | CamelQuarkusMavenInformation | CamelSpringBootMavenInformation | undefined
   >;
+
+  /**
+   * Notifies the host application that a step was added.
+   * @param stepType The type of the step that was added (e.g., "component", "processor", "entity").
+   * @param stepName The name of the step that was added.
+   */
+  onStepAdded(stepType: CatalogKind, stepName: string): Promise<void>;
```

Comment:
Hi @djelinek, we need to implement this method to get notified in VS Code when a new step is added to the canvas. If it helps, it could be implemented temporarily simply to log the added step in the output pane, but ideally, this method should call the update dependencies command.